collect_app/src/androidTest/java/org/odk/collect/android/feature/settings/CustomServerPathsTest.java

```diff
@@ -0,0 +1,101 @@
+package org.odk.collect.android.feature.settings;
+
+import android.Manifest;
+import android.webkit.MimeTypeMap;
+
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import androidx.test.rule.GrantPermissionRule;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.RuleChain;
+import org.junit.runner.RunWith;
+import org.odk.collect.android.R;
+import org.odk.collect.android.injection.config.AppDependencyModule;
+import org.odk.collect.android.openrosa.OpenRosaHttpInterface;
+import org.odk.collect.android.support.CopyFormRule;
+import org.odk.collect.android.support.FeatureTestRule;
+import org.odk.collect.android.support.ResetStateRule;
+import org.odk.collect.android.support.StubOpenRosaServer;
+import org.odk.collect.android.support.pages.GeneralSettingsPage;
+import org.odk.collect.android.support.pages.MainMenuPage;
+import org.odk.collect.utilities.UserAgentProvider;
+
+@RunWith(AndroidJUnit4.class)
+public class CustomServerPathsTest {
```

Comment:
Even though this is now a feature test, I think keeping the link to @opendatakit/testers' internal numbering (NODK-235) would be good.